modules/gapi/include/opencv2/gapi/infer/onnx.hpp

```diff
@@ -277,6 +281,35 @@ template<typename Net> class Params {
     detail::ParamDesc desc;
 };
 
+/*
+* @brief This structure provides functions for generic network type that
+* fill inference parameters.
+* @see struct Generic
+*/
+template<>
+class Params<cv::gapi::Generic> {
```

Comment:
It'd be great to add test for `generic` onnx inference because it becomes core functionality for c++ users.